microsoft-edge/extensions/publish/add-ons-badge.md

```diff
@@ -0,0 +1,81 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 09/15/2025
+---
+# Use the Microsoft Edge Add-ons badge to promote your add-on
+<!-- upstream:
+Microsoft Store marketing guidelines for apps
+https://learn.microsoft.com/windows/apps/publish/app-marketing-guidelines
+https://web.archive.org/web/20250727064821/https://learn.microsoft.com/en-us/windows/apps/publish/app-marketing-guidelines
+not in TOC
+docset: Windows > App development
+-->
+
+To promote your Microsoft Edge extension (or theme), use the Microsoft Edge Add-ons badge in your websites, documentation, tutorials, and other services or integrations.  This ensures consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.
+
+The Microsoft Edge Add-ons badge is a visual asset designed to help you promote your Microsoft Edge add-on and build user trust.
+
+The Microsoft Edge Add-ons badge reads **Get it from Microsoft Edge**, and shows the Microsoft Edge logo:
+
+![The "Microsoft Edge Add-ons" badge](./add-ons-badge-images/microsoft-edge-add-ons-badge.png)
+
+The Microsoft Edge Add-ons badge appears in full color with a dark background, and includes a gray border and transparent corners.  The badge `.png` files are provided in 41 languages.
+
+The Microsoft Edge Add-ons badge lets you showcase your Microsoft Edge add-on with Microsoft Edge-specific branding.  To prevent confusion with the broader [Microsoft Store](https://apps.microsoft.com), the Microsoft Edge Add-ons badge is intended exclusively for promoting add-ons that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser.
+
+The generic Microsoft Store badge reads **Get it from Microsoft**, and shows the Microsoft logo.
+
+Unlike the generic Microsoft Store badge, the Microsoft Edge Add-ons badge provides a distinct, trusted visual cue that connects your Microsoft Edge add-on directly to the Microsoft Edge ecosystem.
+
+The Microsoft Edge Add-ons badge is not the same as the Microsoft brand logo, and they cannot be used interchangeably.
+
+
+<!-- ====================================================================== -->
+## Download the badge image files
+
+Download the complete set of `.png` files: 
+
+* [Microsoft_Edge_Add-ons_badge_image_files.zip](https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/assets/Microsoft_Edge_Add-ons_badge_image_files.zip) (from the [MicrosoftEdge-Extensions](https://github.com/microsoft/MicrosoftEdge-Extensions) repo).
+
+
+<!-- ====================================================================== -->
+## Badge usage guidelines
+
+Follow the required specifications for image usage, fonts, and color palette.  For detailed usage examples and guidelines, download and read the PDF file:
```

Comment:
There are no fonts guidelines below, or in the PDF (I don't think), and you can't change the color within the badge because it's an image. What about:

```suggestion
Follow these guidelines for using the Microsoft Edge Add-ons badge.  For detailed usage examples and guidelines, download and read the PDF file:
```